UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/MetaDataNode.cs

```diff
@@ -32,7 +32,7 @@ public class MetaDataNode : IGasMixContainer
 	/// <summary>
 	/// Used for calculating explosion data
 	/// </summary>
-	public ExplosionNode ExplosionNode = null;
+	public ExplosionNode[] ExplosionNodes = new ExplosionNode[5];
```

Comment:
Why 5? What's this magic number?

Reply:
Bod didnt like list so I replaced it with array. For now 2 is optimal size for it since theres only 2 types of nodes rn, but in case there will be over 5 types it can be expanded

Reply:
Is there a way to cast from the enum? to how many types there are